### ISSUE TITLE
csound~ and Ableton Live 10

### ISSUE THREAD

**Issue by merlinusambrosius** [csound/csound_tilde#1]
I am using Csound 6.07, as csound~ doesn't work with any Csound version after 6.07. Anyways, Live 10 is out now, and the csound~ Max4Live object doesn't seem to work when you add a device to a track. However, when you jump into the editor, it works, but only while in the editor.

**Comment by vlazzarini** [comment]
Do we have a maintainer for this project, or is it orphan?

**Comment by kunstmusik** [comment]
I've announced multiple times over the past few years that I'm no longer
working on this and looked for a new maintainer.  There was one volunteer
but they didn't end up doing any work on it.  So we still need a new
maintainer.

**Comment by vlazzarini** [comment]
Thanks, I think we are clear on this now. I had the impression someone had volunteered.

**Comment by vlazzarini** [comment]
I have sent a request for leaders/maintainers. Until some takes this on, I am afraid we will not
be able to do anything about this issue.

**Comment by merlinusambrosius** [comment]
Thanks.  I'll try to go over the code in the meantime and see if I can make sense of it, and perhaps see if I have what it takes to contribute.

**Comment by vlazzarini** [comment]
I think in the first instance, the code might not need any changes, just need to be built.

**Comment by merlinusambrosius** [comment]
Got it.  Let me see what I can do.

**Comment by merlinusambrosius** [comment]
Alright, after hours of fiddling with unsuccessful building . . . too many errors in Visual Studio to mention . . . I tried copying another copy of the file I had for this into a folder in the Max path, and it seems to have resolved itself.  Apparently the file hosted [here](http://csound.com/download.html) is incorrectly named "csound-" instead of "csound~", and was causing problems.  

Closing, but will continue to try and help out with a build for the latest Csound version for Windows, as time allows.
